crates/iota-sdk-types/src/transaction/serialization.rs

```diff
@@ -772,8 +772,7 @@ mod signed_transaction {
     ///
     /// enum AppId {
```

Comment:
```suggestion
    /// enum IntentAppId {
```
Please also change L766

Reply:
https://github.com/iotaledger/iota-rust-sdk/pull/437/commits/a40819cb416305729de7bc293c003eed3ed38abf